desktop/apps/consignments2/components/step_marker/index.js

```diff
@@ -27,18 +28,48 @@ function StepMarker ({ currentStep, isMobile, steps }) {
   )
 }
 
-const mapStateToProps = (state) => ({
-  currentStep: state.submissionFlow.currentStep,
-  isMobile: state.submissionFlow.isMobile,
-  steps: state.submissionFlow.steps
-})
+const mapStateToProps = (state) => {
+  const {
+    submissionFlow: {
+      currentStep,
+      isMobile,
+      steps
+    }
+  } = state
+
+  const stepLabels = {
+    createAccount: {
+      label: 'Create Account',
+      shortLabel: 'Create'
+    },
+    chooseArtist: {
+      label: 'Consign Artist/Designer',
+      shortLabel: 'Consign'
+    },
+    describeWork: {
+      label: 'Describe the Work',
+      shortLabel: 'Describe'
+    },
+    uploadPhotos: {
+      label: 'Upload Photos',
+      shortLabel: 'Upload'
+    }
```

Comment:
Not a blocker, but I would considering putting this into a generic configuration map and including a `url` field that corresponds to the appropriate step, then passing it around like 

```javascript
const steps = {
  createAccount: {
    label: 'Create Account',
    shortLabel: 'create',
    url: '/consign2/create'
  },
  ...
}

// later...
<Route path={steps.createAccount.url} />
```